apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -276,6 +276,52 @@ module.exports = {
             },
           ],
         },
+        {
+          label: "Imprint",
+          name: "imprint",
+          widget: "object",
+          file: "content/pages/imprint.md",
+          fields: [
+            {
+              label: "Title",
+              name: "title",
+              widget: "string",
+            },
+            {
+              label: "Subtitle",
+              name: "subtitle",
+              widget: "string",
+            },
+            {
+              label: "Content",
+              name: "content",
```

Comment:
```suggestion
              label: "Body",
              name: "body",
```

This way body will be converted to `content` when we call `matter` function (label can remain `Content` if it make more sense).